projects/ngx-editor/src/lib/defaultPlugins.ts

```diff
@@ -19,7 +19,7 @@ interface ShortcutOptions {
   history: boolean;
 }
 
-const isMacOs = /Mac/.test(navigator.platform);
+const isMacOs = typeof navigator !== 'undefined' ? /Mac/.test(navigator.platform) : false
```

Comment:
```suggestion
const isMacOs = /Mac/.test(navigator?.platform);
```

Let's keep it simple.

Reply:
i guess they are different
cause your suggestion will generate
`var isMacOs = /Mac/.test(navigator === null || navigator === void 0 ? void 0 : navigator.platform);`

but the navigator won't be defined, it will face same error. (navigator is not defined)

Reply:
My bad. I was just trying to keep it simple and forgot that it will be undefined at the first place.